app/src/main/java/cz/muni/fi/pv256/movio2/uco_410371/dummy/DummyContent.java

```diff
@@ -0,0 +1,32 @@
+package cz.muni.fi.pv256.movio2.uco_410371.dummy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import cz.muni.fi.pv256.movio2.uco_410371.models.Movie;
+
+/**
+ * Dummy data
+ * Created by Benjamin Varga on 6.10.2016.
+ */
+
+public class DummyContent {
+
+    public static final List<Movie> MOVIES = new ArrayList<>();
+    public static final List<Object> ITEMS = new ArrayList<>();
```

Comment:
Do takehoto listu by mohol niekto pridat aj objekty, ktore tam nepatria (napr. Book, Integer...cokolvek).
Treba to zamedzit bud Javadoc komentarom, alebo nejakym spolocnym typom.